newIDE/app/src/UI/BackgroundText.js

```diff
@@ -4,7 +4,7 @@ import ThemeConsumer from './Theme/ThemeConsumer';
 
 const styles = {
   messageStyle: {
-    opacity: 0.4,
+    //opacity: 0.4,
```

Comment:
Can we still keep a bit of opacity? The idea is that these messages should not be too disturbing to the user - they are not "must read" messages but only something to hint you about doing something when you're lost for the first time.
So they should feel part of the background.

In a way, it *is* done on purpose that they are hard to read. As a new user, you're starring at the screen and will read them to understand what's going on.
As an experienced user, you want these to feel invisible and don't disturb your workflow.

Reply:
Ok i will uncomment this line :)